vulyk/bootstrap/__init__.py

```diff
@@ -51,8 +51,26 @@ def init_app(name):
         _assets.init(app)
         _social_login.init_social_login(app, db)
 
-        app.register_blueprint(gamification, url_prefix='/gamification')
-        app.logger.info('Gamification module loaded.')
+        enabled_blueprints = app.config.get('ENABLED_BLUEPRINTS', [])
```

Comment:
Please, extract the code to separate file. This makes code as declarative as it could be.